atm.js

```diff
@@ -1,21 +1,145 @@
+    balCheck : $("#balCheck")
+  };
 }
+var checkReg = new Checkings();
 
-function checkingWithdrawal(amount){
+//Calculate total of Checkings Deposits
+Checkings.prototype.calculateTotal = function(){
+  var total = 0;
+  for( var i = 0; i < this.entries.length; i++ ){
+    total += this.entries[i];
+  }
+  return total;
+};
 
-}
+//When deposit button is clicked, add value to an array
+depCheck.on("click", function(event) {
+  event.preventDefault();
+  var userInput = parseFloat(checkReg.els.entCheck.val());
+  checkReg.entries.push(userInput);
+  //calculate total
+  var newTotal = checkReg.calculateTotal();
+  //update html for the total
+  checkReg.els.balCheck.html("$" + newTotal.toFixed(2));
+  //clear the input
+  checkReg.els.entCheck.val("");
+  makeCheckRed();
+});
 
-function savingsDeposit(amount){
+//When withdraw button is clicked, subtract value from sum of arrays
+witCheck.on("click", function(event) {
+  event.preventDefault();
+  //Make value a negative number and push to entries array
+  var userInput = -Math.abs(checkReg.els.entCheck.val());
+  checkReg.entries.push(userInput);
+  //calculate total
+  var newTotal = checkReg.calculateTotal();
+  if (newTotal >= 0) {
+    //update html for the total
+    checkReg.els.balCheck.html("$" + newTotal.toFixed(2));
+  } else {
+    //prevent overdraft
+    checkReg.entries.pop();
+    console.log(checkReg.entries);
+  }
+  //clear the input
+  checkReg.els.entCheck.val("");
+  makeCheckRed();
+});
 
+//add class of zero when balance is zero
+function makeCheckRed() {
+  if (checkReg.els.balCheck.html() === "$0.00") {
+    checkReg.els.balCheck.addClass("zero");
+  } else {
+    checkReg.els.balCheck.removeClass("zero");
+  }
 }
+makeCheckRed();
 
-function savingsWithdrawal(amount){
+////////////
+//Savings//
+///////////
 
+//Create Savings Constructor
+function Savings(){
+  this.entries = [];
+  this.els = {
+    //input text
+    entSave : $("#entSave"),
+    //total money
+    balSave : $("#balSave")
+  };
 }
+var saveReg = new Savings();
+
+//Calculate total of Savings Deposits
+Savings.prototype.calculateTotal = function(){
+  var total = 0;
+  for( var i = 0; i < this.entries.length; i++ ){
+    total += this.entries[i];
+  }
+  return total;
+};
+
+//When deposit button is clicked, add value to an array
+depSave.on("click", function(event) {
+  event.preventDefault();
+  var userInput = parseFloat(saveReg.els.entSave.val());
+  saveReg.entries.push(userInput);
+  //calculate total
+  var newTotal = saveReg.calculateTotal();
+  //update html for the total
+  saveReg.els.balSave.html("$" + newTotal.toFixed(2));
+  //clear the input
+  saveReg.els.entSave.val("");
+  makeSaveRed();
+});
 
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+//When withdraw button is clicked, subtract value from sum of arrays
+witSave.on("click", function(event) {
+  event.preventDefault();
+  //Make value a negative number and push to entries array
```

Comment:
This click event looks really similar to the above Checking one. I think you could create one callback function for both